src/main/java/Duke.java

```diff
@@ -1,10 +1,93 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
+        int count = 0;
+        String input;
+        Task list[] = new Task[100];
+
+        printWelcomeMessage();
+
+        // Echo loops until "bye" is inputted
+        while(true) {
+            Scanner echo = new Scanner(System.in); // Scan and echo user input
+            input = echo.nextLine();
+
+            // Exit program if "bye" is inputted
+            if (input.equals("bye")) {
+                System.out.println("Goodbye. See you soon!");
```

Comment:
Perhaps can extract this to a method outside of the while loop so that the level of abstraction inside the while loop is consistent.